src/main/java/com/sun/tdk/signaturetest/core/ClassCorrector.java

```diff
@@ -56,7 +56,12 @@
 
     protected ClassHierarchy classHierarchy = null;
     private Log log;
-
+    private JDKExclude jdkExclude = new JDKExclude() {
+        @Override
+        public boolean isJdkClass(String name) {
+            return false;
+        }
+    };
```

Comment:
This is just a nit, but if you have the existing constructor call your new overloaded one passing this in, you can make this final and have the new constructor do a null check on the jdkExclude param.

Looking at this I was concerned about NPEs when this field is used.